sdk/OoyalaSkinSDK/discoveryPanel.js

```diff
@@ -99,6 +135,16 @@ var DiscoveryPanel = React.createClass({
     );
   },
 
+  renderCircularStatus: function() {
+    return (
+      <CircularStatus
+        onPress={() => this.onStatusPressed()}
+        total={this.state.counterLimit}
+        current={this.state.currentCounterVal}
+        thickness={2}
+        diameter={44} />);
```

Comment:
I agree, but where could be a good place for they to live? Within constants? In the skin-config?